packages/Frontend/src/containers/AppContainer.jsx

```diff
@@ -1,25 +1,17 @@
 import React, { Component } from 'react';
-import Amplify from 'aws-amplify';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { BrowserRouter } from 'react-router-dom';
+import { auth } from 'firebase';
 import App from '../components/App';
 import logout from '../actions/logout';
-
```

Comment:
This is how we configured Amazon before to point to our account information, no longer needed.